lib/generators/talking_stick/views/views_generator.rb

```diff
@@ -0,0 +1,30 @@
+class TalkingStick::ViewsGenerator < Rails::Generators::Base
+  source_root File.expand_path('../templates', __FILE__)
+
+  def generate_participant_views
+    copy_file "#{copy_path}/participants/_form.html.erb", "#{paste_path}/participants/_form.html.erb"
+    copy_file "#{copy_path}/participants/edit.html.erb", "#{paste_path}/participants/edit.html.erb"
+    copy_file "#{copy_path}/participants/index.html.erb", "#{paste_path}/participants/index.html.erb"
+    copy_file "#{copy_path}/participants/new.html.erb", "#{paste_path}/participants/new.html.erb"
+    copy_file "#{copy_path}/participants/show.html.erb", "#{paste_path}/participants/show.html.erb"
+  end
+
+  def generate_room_views
+    copy_file "#{copy_path}/rooms/_form.html.erb", "#{paste_path}/rooms/_form.html.erb"
```

Comment:
@benlangfeld well, I can, but I need to figure out how to do it.  do you want me to switch gears and start working on this?